code/modules/surgery/organs/subtypes/grey.dm

```diff
@@ -1,3 +1,17 @@
 /obj/item/organ/internal/liver/grey
 	alcohol_intensity = 1.6
 	species = "Grey"
+
+/obj/item/organ/internal/brain/grey
+	icon_state = "brain-x"
+	mmi_icon_state = "mmi_alien"
+	species = "Grey"
+
+/obj/item/organ/internal/brain/grey/insert(var/mob/living/carbon/M, var/special = 0)
+	..()
+	M.add_language("Psionic Communication")
```

Comment:
Won't they need to have a Grey's brain to do this anyways? Since a brain is as tied to an identity as you can get, I'm not sure how this could be abused